transaction-status/src/parse_token/extension/confidential_transfer.rs

```diff
@@ -34,27 +35,24 @@ pub(in crate::parse_token) fn parse_confidential_transfer_instruction(
             })
         }
         ConfidentialTransferInstruction::UpdateMint => {
```

Comment:
I can't really comment on non-changed part of the code, but for `InitializeMint` instruction data, we should remove `mint` and add `confidentialTransferMintAuthority,`, `autoApproveNewAccounts`, and `auditorElGamalPubkey`. Sorry I missed this on my previous review.

Reply:
Will do

Reply:
Done in 0d50498a99